content/participants/ttjc@prernamehraa/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Prerna Mehra
+type: "participant"
+name: "Prerna mehra"
+portfolio: "http://prernamehraa.pythonanywhere.com/"
+github-link: "https://github.com/sadakchap"
+linkedin-link: "https://linkedin.com/in/prerna-mehra"
+looking-for: "job"
+description: "Python | Django-REST-APIS | React"
+---
+
+Hi! I am a 3rd year B.Sc. Student, currently studing react and UI.
+### introduction
+I am from Chhindwara, Madhya Pradesh, India.
+
+
+## why are you taking this challenge?
+
+I need an job in December 2020.
+I want to learn becoz learning makes me feel alive😊.
+I want to work with mentors.
+
+## something interesting about you?
+I am patient, but sometimes I am late too.
+I still remember all of the roads of GTA-vice city and still think that "helicopter" mission is the hardest one!
+
+### help I need?
+
+It would be good if someone tells me more about react state, still confused in it.
+
+### how to contact me?
+
+- [Twitter](#)
```

Comment:
@sadakchap , can you add your twitter handle here or remove the section if you don't have any.